pkg/minishift/config/config.go

```diff
@@ -27,12 +27,14 @@ var Config *InstanceConfig
 type InstanceConfig struct {
 	FilePath string `json:"-"`
 	OcPath   string
+
+	HostFolders []HostFolder
 }
 
 // Create new object with data if file exists or
 // Create json file and return object if doesn't exists
 func NewInstanceConfig(path string) (*InstanceConfig, error) {
-	cfg := new(InstanceConfig)
+	cfg := &InstanceConfig{HostFolders: []HostFolder{}}
```

Comment:
Yes, IMO it is instance specific. making it part of ~/.minishift/config/config.json will hinder the multiple instance story.

I might able to mount different folder to different instances. `many-many relationship` here.